test/elm/clinical/clinical-test.ts

```diff
@@ -46,6 +46,27 @@ describe('ValueSetRef', () => {
   });
 });
 
+describe('AnyInValueSet', () => {
+  beforeEach(function () {
+    setup(this, data, [], vsets);
+  });
+
+  it('should call execute on codes which is a list of concepts', async function () {
+    (await this.anyInListOfConcepts.exec(this.ctx)).should.be.true();
+    should(this.ctx.localId_context[12]).not.be.undefined();
```

Comment:
Good call! This will be easy to fix- thank you!

Reply:
@elsaperelli - I apologize for not calling it out explicitly, but quite a few local ids were also added in the `list-test.ts` and `query-test.ts` files as well... This is why I said it would be pretty annoying to fix at scale -- because, dang, it _will_ be.

The tests work now -- it's more about preventing unintended surprises in the future. So if we want to hold off on updating the rest of those tests so we can get this PR out the door, I totally understand.